### ISSUE TITLE
[Accessibility][Info&Relationship-Filter Dialog]: Aria label and place holder both are defined same in the code on 'Filter Dialog' edit box.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2054]
**User Experience** 
Screen reader users will get confused if the same information is convey by the narrator twice.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE and set the zoom at 400%.
2. Select 'Design Flow' from left navigation.
3. Now, navigate to the filter dialog edit field.
4. Observe the narrator narration and find the issue.

**Actual:** 
Aria label and place holder both are defined same in the code on 'Filter Dialog' edit box due to which screen reader is conveying the same information twice.

**Expected:**
Aria label and place holder both should be defined differently so that same information will not be convey twice. In Aria Label it should be 'Search available filters'.

**Observation:**
Same issue observed with NVDA.

**MAS Impacted:** [MAS1.3.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4})

**Attachment:**
[MAS1.3.1_Aria lable and placeholder both are same.zip](https://github.com/microsoft/BotFramework-Composer/files/4240933/MAS1.3.1_Aria.lable.and.placeholder.both.are.same.zip)

**Comment by beyackle** [comment]
Is this problem now fully fixed and we can move this into Waiting for Verification, or is there more?

**Comment by ashish315** [comment]
As checked now the aria label and placeholder are different but due to "placeholder narrated twice" know issue it still announces placeholder twice already raised issue for this.
So, HCL is closing this issue.
